src/components/drag-and-drop/drag-and-drop.stories.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
```

Comment:
This formatting, i.e. the subtitles as set against the title-sized text, doesn't seem right. Is there a way to give the first subtitle more breathing space?

![image](https://user-images.githubusercontent.com/26925206/53957570-91837000-40d6-11e9-91ed-65c7b4c5e765.png)